src/app/page.tsx

```diff
@@ -1,34 +1,29 @@
 "use client";
 
 import Card from "./_components/Card";
-// import {
-//   Form,
-//   FormControl,
-//   FormDescription,
-//   FormField,
-//   FormItem,
-//   FormLabel,
-//   FormMessage,
-// } from "./_components/ui/form";
-// import { Input } from "./_components/ui/input";
+import { api } from "~/trpc/react";
 
 // import { useCountStore } from "~/store/store";
 
 export default function Home() {
   // const counter = useCountStore((state) => state.counter);
   // const increase = useCountStore((state) => state.increase);
   // const form = useForm();
+  // const { data } = api.categories.greeting.useQuery();
+
+  // const { data: recipes } = api.categoriesRouter.getCategory.useQuery();
+  const { data: recipeData, isLoading} = api.recipesRouter.getSortedRecipes.useQuery();
+
+  if (recipeData) {
+    console.log(recipeData);
+  }
 
   return (
     <main className="flex min-h-screen flex-col items-center justify-center text-white">
-      {/* <Button variant="ghost">Button</Button> */}
       {/* <h1>Books: {counter} </h1> */}
-      <div className="grid grid-cols-4 gap-5">
-        <Card />
-        <Card />
-        <Card />
-        <Card />
-        <Card />
+      <div className="grid grid-cols-2 gap-5">
+        {!isLoading ?
```

Comment:
try to use useSuspenseQuery and add loading.tsx file

Reply:
https://recipes-il7ijkxaq-ihors-projects-85c75ba3.vercel.app/